core/src/test/kotlin/ch/ergon/dope/resolvable/expression/AsteriskExpressionTest.kt

```diff
@@ -0,0 +1,42 @@
+package ch.ergon.dope.resolvable.expression
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.helper.someBucket
+import ch.ergon.dope.resolvable.expression.unaliased.type.ParameterManager
+import org.junit.jupiter.api.BeforeEach
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class AsteriskExpressionTest {
+
+    @BeforeEach
+    fun setUp() {
+        ParameterManager.resetCounter()
+    }
```

Comment:
I see you use this kind of setup in a few places, maybe we could add this to a more general test class?

Reply:
done